internal/ws/wsserver_test.go

```diff
@@ -192,16 +192,27 @@ func TestTimeoutProcessing(t *testing.T) {
 	assert.NoError(err)
 
 	c.WriteJSON(&webSocketCommandMessage{
-		Type: "ack",
+		Type:  "ack",
+		Topic: "mytopic",
+	})
+	c.WriteJSON(&webSocketCommandMessage{
+		Type:  "ack",
+		Topic: "mytopic",
 	})
+	c.Close()
 
-	// Confirm we close the connection after the timeout pops
 	for len(w.connections) > 0 {
```

Comment:
This loop can be dropped now, right?

Reply:
Or maybe you are waiting for w to close after c is closed...

Reply:
Agreed - it's actually the second loop that needs to be dropped though 👍